.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
```

Comment:
Awesome, these were the exact params I've been using for almost the past year using prettier 👍 

<img width="1083" alt="screen shot 2018-02-12 at 9 59 56 am" src="https://user-images.githubusercontent.com/5067716/36102838-8c34494e-0fdb-11e8-816a-6326d0fd42c4.png">

```
/usr/local/lib/node_modules/prettier/bin/prettier.js --write --single-quote=true --trailing-comma=all --print-width=100 src/client/app/appReducer.js
```